pkg/tsdb/cloudwatch/constants/metrics.go

```diff
@@ -449,27 +449,33 @@ var NamespaceMetricsMap = map[string][]string{
 	"AWS/SNS":                  {"NumberOfMessagesPublished", "NumberOfNotificationsDelivered", "NumberOfNotificationsFailed", "NumberOfNotificationsFilteredOut", "NumberOfNotificationsFilteredOut-InvalidAttributes", "NumberOfNotificationsFilteredOut-NoMessageAttributes", "PublishSize", "SMSMonthToDateSpentUSD", "SMSSuccessRate"},
 	"AWS/SQS":                  {"ApproximateAgeOfOldestMessage", "ApproximateNumberOfMessagesDelayed", "ApproximateNumberOfMessagesNotVisible", "ApproximateNumberOfMessagesVisible", "NumberOfEmptyReceives", "NumberOfMessagesDeleted", "NumberOfMessagesReceived", "NumberOfMessagesSent", "SentMessageSize"},
 	"AWS/SWF":                  {"ActivityTaskScheduleToCloseTime", "ActivityTaskScheduleToStartTime", "ActivityTaskStartToCloseTime", "ActivityTasksCanceled", "ActivityTasksCompleted", "ActivityTasksFailed", "ConsumedCapacity", "DecisionTaskScheduleToStartTime", "DecisionTaskStartToCloseTime", "DecisionTasksCompleted", "PendingTasks", "ProvisionedBucketSize", "ProvisionedRefillRate", "ScheduledActivityTasksTimedOutOnClose", "ScheduledActivityTasksTimedOutOnStart", "StartedActivityTasksTimedOutOnClose", "StartedActivityTasksTimedOutOnHeartbeat", "StartedDecisionTasksTimedOutOnClose", "ThrottledEvents", "WorkflowStartToCloseTime", "WorkflowsCanceled", "WorkflowsCompleted", "WorkflowsContinuedAsNew", "WorkflowsFailed", "WorkflowsTerminated", "WorkflowsTimedOut"},
-	"AWS/SageMaker":            {"ClientInvocations", "ClientInvocationErrors", "ClientLatency", "CPUUtilization", "DatasetObjectsAutoAnnotated", "DatasetObjectsHumanAnnotated", "DatasetObjectsLabelingFailed", "DiskUtilization", "ExecutionStarted", "ExecutionFailed", "ExecutionSucceeded", "ExecutionStopped", "ExecutionDuration", "GPUMemoryUtilization", "GPUUtilization", "Invocation4XXErrors", "Invocation5XXErrors", "InvocationModelErrors", "Invocations", "InvocationsPerCopy", "InvocationsPerInstance", "JobsFailed", "JobsStopped", "JobsSucceeded", "LoadedModelCount", "MemoryUtilization", "ModelCacheHit", "ModelDownloadingTime", "ModelLatency", "ModelLoadingTime", "ModelLoadingWaitTime", "ModelSetupTime", "ModelUnloadingTime", "NumberOfUsers", "OverheadLatency", "StepStarted", "StepFailed", "StepSucceeded", "StepStopped", "StepDuration", "TotalDatasetObjectsLabeled"},
```

Comment:
Can you copy paste the [old SageMaker metrics and dimensions](https://github.com/grafana/grafana/pull/85009/files) and dimensions from before the last pr to the new one? It seems like the [Ground Truth Metrics](https://docs.aws.amazon.com/sagemaker/latest/dg/monitoring-cloudwatch.html#cloudwatch-metrics-ground-truth) from the docs that were in the old SageMaker metrics are missing.

Reply:
whoops, thank you added them back in

Reply:
added InferenceComponents metrics as well

Reply:
to be clear, for GroundTruth, the correct namespace is actually `AWS/SageMaker/LabelingJobs` and not `AWS/SageMaker`